source/docs/software/driverstation/driver-station-best-practices.rst

```diff
@@ -1,77 +1,97 @@
+    a. A built-in Ethernet port is highly preferred. Ensure that it's a full-sized port. The hinged Ethernet ports don't hold up to repeated use.
+    #. If the Ethernet port on your laptop is dodgy, either replace the laptop (recommended) or buy a USB Ethernet dongle from a reputable brand. Many teams find that USB Ethernet is less reliable than built-in Ethernet, primarily due to cheap hardware and bad drivers. The dongles given to rookies in the KOP have a reputation for working well.
+    #. 2 USB ports minimum.  These can be USB-A or USB-C ports, depending on the peripherals you're using.
 
-5. Install all Windows updates a week before the competition. This allows you time to ensure the updates will not interfere with driver station functions. To do so, open the Windows Update settings page and see that you're up-to-date. Install pending updates if not. Reboot and check again to make sure you’re up to date.
-6. Change "Active Hours" for Windows Updates to prevent updates from installing during competition hours. Navigate to Start -> Settings -> Update & Security -> Windows Update, then select Change active hours. If you’re traveling to a competition, take time zone differences into account. This will help ensure your driver station does not reboot or fail due to update installing on the field.
-7. Remove any 3rd party antivirus or antimalware software. Instead, use Windows Defender on Windows 10 or 11. Since you're only connecting to the internet for Windows and FRC software updating, the risk is low. Only install software on your driver station that's needed for driving. Your goal here is to eliminate variables that might interfere with proper operation. Remove any unneeded preinstalled software ("bloatware") that came with the machine. Don't use the laptop as your Steam machine for gaming back at the hotel the night before the event. Many teams go as far as having a separate programming laptop.
-8. Avoid managed Windows 10 or 11 installations from the school's IT department. These deployments are built for the school environment and often come with unwanted software that interferes with your robot's operation.
-9. Laptop battery / power
+#.  A keyboard. It’s hard to quickly do troubleshooting on touch-only computers at the field.
+#.  A solid-state disk (SSD), 256 GB or larger. If the laptop has a rotating disk, spend $50 and replace it with a SSD.
+#.  Supports Windows 11.  While Windows 10 is still supported for driver station use, acquiring a machine that can't run Windows 11 isn't forward thinking.
+#.  AMD or Intel CPU. The Driver Station software does not work on machines with ARM CPUs.
+#.  2.4 and 5 GHz Wi-Fi radio, with 6GHz optional.  The `recommended practice configuration <https://frc-radio.vivid-hosting.net/overview/practicing-at-home#vh-109-as-an-access-point-recommended>`__ does not require a 6GHz radio in the driver station laptop.
```

Comment:
Well, 2.4 and 5ghz wifi aren't needed either :-)

In my case, if I was buying a brand new $1000+ laptop, having the flexibility to use it wirelessly would be worth it to me. If I was buying a $300 used laptop, I wouldn't bother. I might consider a pro/con (pro more flexible, con less like field setup) to help teams make their own decision.

The FIRST laptop recommendations list Wifi 6E as "preferred". https://www.firstinspires.org/hubfs/web/program/all/computer-requirements.pdf?hsLang=en